plaso/parsers/apache_access.py

```diff
@@ -54,24 +54,17 @@ class ApacheAccessParser(text_parser.PyparsingSingleLineTextParser):
   NAME = 'apache_access'
   DESCRIPTION = 'Apache access Parser'
 
+  MAX_LINE_LENGTH = 2048
+
   _PYPARSING_COMPONENTS = {
```

Comment:
Please keep the pyparsing components here, to be consistent with other pyparsing parsers.

Reply:
Why? this is addition indirection, which appears to not be used for any other purpose

Reply:
To be consistent with the other pyparsing parsers. If you don't like the pyparsing_components constant, why are remote_name and user_name still defined there?

Reply:
Because I did not had to touch them. I can change them. Agree that consistency is important but ` _PYPARSING_COMPONENTS` was never was introduces across all pyparsing based parsers e.g. https://github.com/log2timeline/plaso/blob/master/plaso/parsers/xchatlog.py

If you say this indirection is not used I'll create an issue to remove it everywhere.